run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+chown -R bitcoingold:bitcoingold /home/bitcoingold/
```

Comment:
The way the chown is running at the moment is it'll be run as the user `bitcoingold`. Should the directory not be owned by `bitcoingold` that command will fail. Additionally, it's better to simply add it the Dockerfile as another layer just before `USER`. No need for an extra file.

Reply:
the run file it's because on startup it update right ("chown") and when we use a volume to store data it's useful

Reply:
But if you don't have the rights to that directory (and you won't if it's not root), then surely that command will just fail and not make a difference?